src/main/java/com/PBA/authservice/exceptions/AuthValidationException.java

```diff
@@ -0,0 +1,19 @@
+package com.pba.authservice.exceptions;
+
+import java.util.Map;
+
+public class AuthValidationException extends RuntimeException {
+    private Map<String, String> errorMap;
```

Comment:
You should have an additional 'code' field here only that can be used in the UserControllerExceptionHandler.